scripts/winesapos-install.sh

```diff
@@ -787,6 +789,18 @@ elif [[ "${WINESAPOS_DE}" == "gnome" ]]; then
         pacman_install_chroot manjaro-gnome-settings manjaro-settings-manager
     fi
     echo "Installing the GNOME desktop environment complete."
+elif [[ "${WINESAPOS_DE}" == "i3" ]]
+      echo "Installing i3"
+    pacman_install_chroot i3-wm xorg xorg-server i3lock i3blocks i3status xcb-util-keysyms xcb-util-cursor xcb-util-wm xcb-util-xrm
+    pacman_install_chroot nano vi vim
+    echo "Installing i3 ... Completed successfully"
+    
+elif [[ "${WINESAPOS_DE}" == "sway" ]]; then
+    echo "Installing the Sway tiling manager..."
+    pacman_install_chroot swaylock swayidle swaybg xorg-server xorg xcb-util-keysyms xcb-util-cursor xcb-util-wm xcb-util-xrm
```

Comment:
The actual `sway` package is missing here.

You also don't need these Xorg dependencies (Sway only works with Wayland and we install Wayland dependencies earlier on).